### DISCUSSION TITLE
로비 -> 타운홀 문으로 이동시 캐릭터 바뀜 현상

### DISCUSSION THREAD

**Discussion by bboddoFactory** [metamong-friends/metamong-friends.github.io#58] [제보하기]
안녕하세요!!

메타몽을 처음 설치하고 캐릭터를 설정한 후, 로비를 돌아다니다가 타운 홀 문으로 들어갔는데, 캐릭터가 다른 것으로 바뀌는 현상을 발견했습니다!!  

혹시 몰라 다른 캐릭터도 선택해서 똑같은 테스트를 했는데, 역시 캐릭터가 바뀝니다!! 
그런데  바뀌는 캐릭터는 다 똑같습니다!! ( 분홍 맨투맨, 파랑 조끼, 검은 바지 캐릭터) 

확인해주시면 감사하겠습니다!! 😊

**Comment by shlee303** [comment]
@yhyuntak 코멘트 감사합니다!!! 저희가 사무실/타운홀이 별도의 스페이스로 구분되어 있고, 스페이스별로 캐릭터/닉네임을 설정 할 수 있는 로직이기 때문에 타운홀 이동 시 이전에 (마지막에) 타운홀 들어가실 때 선택한 캐릭터/닉네임으로 보이게 되는 현상입니다!
포털로 이동하여 동일 캐릭터가 가는 느낌을 받으실 수 있을 것 같네요. 이 부분은 개선 해보도록 하겠습니다!!!!

**Comment by ACT-TJ** [comment]
Space 별 다른 캐릭터, 닉네임을 가져가는 컨셉을 가져가 보았습니다.
Hall Gate 를 통한 직접 이동은 캐릭터, 닉네임 유지하는 부분에 대해 검토해 보겠습니다.
( 장단점이 있을 것 같은데 취지는 충분히 공감 하였습니다! )

제보 너무 감사합니다~!!
